model/states.go

```diff
@@ -139,8 +140,8 @@ type OperationState struct {
 
 // OperationStateTimeout ...
 type OperationStateTimeout struct {
-	StateExecTimeout  StateExecTimeout `json:"stateExecTimeout,omitempty"`
-	ActionExecTimeout string           `json:"actionExecTimeout,omitempty" validate:"omitempty,min=1"`
+	StateExecTimeout  *StateExecTimeout `json:"stateExecTimeout,omitempty"`
```

Comment:
maybe you can revert this change as the timeouts will be fixed as part of https://github.com/serverlessworkflow/sdk-go/pull/104